docs/changelog/85206.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 85206
```

Comment:
I don't think this file is needed anymore.

Reply:
👍 removed